code/mod_table/mod_table.cpp

```diff
@@ -943,6 +962,8 @@ void mod_table_reset()
 	Beams_use_damage_factors = false;
 	Generic_pain_flash_factor = 1.0f;
 	Shield_pain_flash_factor = 0.0f;
+	Emp_pain_flash_factor = 1.0f;
+	Emp_pain_flash_color = std::make_tuple(static_cast<ubyte>(255), static_cast<ubyte>(255), static_cast<ubyte>(127));
```

Comment:
You might want to do a small optimization here and make these all floats from 0 to 1.  Since that is their use case, it makes sense to convert them when they are initialized and parsed, not every time `game_flash` is called.

Reply:
Good idea, changed to floats instead of ubtyes.